### ISSUE TITLE
try to read tag to play list, it doesn't work.

### ISSUE THREAD

**Issue by dg9999cn** [confirm/mopidy-pummeluff#9]
try to read tag to play list, it doesn't work.

**Comment by domibarton** [comment]
What do you mean by _it doesn't work_?

**Comment by domibarton** [comment]
@dg9999cn This should be fixed with `v2.0.0`.

You can either try `master`, the [v2.0.0 release / tag](https://github.com/confirm/mopidy-pummeluff/releases/tag/v2.0.0) or the [official PyPi / pip package](https://pypi.org/project/Mopidy-Pummeluff/2.0.0/).

Let me know if it works!

I had to change something in the management of the registered tags. Was a bit a "messy" code, so I cleaned it up a bit. Unfortunately, it also means the old JSON with the registered tags will no longer be valid. In case you already had some tags registered before, make sure you remove the old JSON before starting Mopidy:

```
# either remove
rm /var/lib/mopidy/pummeluff/tags.json

# or backup
mv /var/lib/mopidy/pummeluff/tags.json /var/lib/mopidy/pummeluff/tags.json.old
```

**Comment by domibarton** [comment]
@dg9999cn closing the issue. The new version works (see #10).
